src/global/responses/errors.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class ConflictErrorResponse {
```

Comment:
Is this a universal error for all modules? The example message may not be generic enough if so. Alternatively maybe add nestJS and team to the class name so it doesn't get used for like a querying conflict or module other than teams.